src/modules/Reviews/interface/ReviewRepository.ts

```diff
@@ -0,0 +1,9 @@
+// src/interfaces/ReviewRepository.ts
+import Review from '../Review'
+
+interface ReviewRepository {
+  findByUserId(userId: string): Promise<Review[]>
+  create(review: Review): Promise<void>
```

Comment:
Is it ok to return void or are we supposed to return something for successful creation?